components/fxa-client/src/oauth.rs

```diff
@@ -302,6 +350,21 @@ impl std::fmt::Debug for AccessTokenInfo {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize)]
+pub struct IntrospectInfo {
+    pub active: bool,
+    pub token_type: String
+}
+
+impl std::fmt::Debug for IntrospectInfo {
```

Comment:
Can't we just derive `Debug` automatically?

Reply:
To add a bit of context: some structs implement their own version of `std::fmt::Debug` in order to avoid accidentally printing secret keys, passwords or similar in log output. For structs that do not contain any PII or confidential data, using `#[derive(Debug)]` is generally fine.

Reply:
What about clientID? It's a field given in the response of introspect.

Reply:
We already know the ClientID when we create a FirefoxAccount struct so I guess you can ignore the field.

Reply:
`client_id` is safe to log, it just identifies the software that you're running (e.g. there's one for Fenix, one for Firefox iOS, etc).